package.json

```diff
@@ -90,5 +90,9 @@
   },
   "browserslist": [
     "extends browserslist-config-kolibri"
-  ]
+  ],
+  "volta": {
```

Comment:
I'm not sure about the raw loader upgrade, but I think this volta was accidentally committed from your local environment (it has happened to me.... quite a few times 😄)

Reply:
The `raw-loader` was indeed intensional! I have synced the codebase with the upstream, so the Volta diff is not resolved :)